lacework/resource_lacework_resource_group_aws.go

```diff
@@ -0,0 +1,192 @@
+package lacework
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"log"
+	"strings"
+
+	"github.com/lacework/go-sdk/api"
+)
+
+func resourceLaceworkResourceGroupAws() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceLaceworkResourceGroupAwsCreate,
+		Read:   resourceLaceworkResourceGroupAwsRead,
+		Update: resourceLaceworkResourceGroupAwsUpdate,
+		Delete: resourceLaceworkResourceGroupAwsDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: importLaceworkIntegration,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The resource group name",
+			},
+			"enabled": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     true,
+				Description: "The state of the external integration",
+			},
+			"description": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The description of the resource group",
+			},
+			"accounts": {
+				Type: schema.TypeList,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+					StateFunc: func(val interface{}) string {
+						return strings.TrimSpace(val.(string))
+					},
+				},
+				Required:    true,
+				Description: "The list of aws accounts to include in the resource group",
+			},
+			"id": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The resource group unique identifier",
+			},
+			"lacework_account_id": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The lacework account id",
+			},
+			"last_updated": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The time in millis when the resource was last updated",
+			},
+			"updated_by": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The username of the lacework user who performed the last update",
+			},
+			"type_name": {
```

Comment:
Let us call this computed attribute just `type`. We enherated `type_name` from APIv1.
```suggestion
			"type": {
```